iroh-util/src/exitcodes.rs

```diff
@@ -0,0 +1,23 @@
+//! iroh exit codes
```

Comment:
could also move this into an errors package. I've just crammed it here for now, and don't think it's a big deal to move later once we start getting more rigorous about error categorization